### PR TITLE
Feat/get transaction

### PR DESCRIPTION
**Pull request by tobi-bams**

### CODE REVIEW

File: src/server/routes/transaction.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                return res.status(200).json({
+                    status: "ok",
+                    message: "all transaction",
+                    nextPage: transactions.nextPage,
+                    currentPage: transactions.currentPage,
+                    remainingPage: transactions.remainingPages,
+                    data: transactions.transactions
+                })
+            } catch (error) {
+                Logger.error(`Error in quote final: ${error}`)
```
Comment (Extheoisah) [inline src/server/routes/transaction.ts:67]:
```suggestion
                Logger.error(`Error fetching transactions: ${error}`)
```

File: src/services/sequelize/repositories/transaction.ts [hunk 2]
```diff
@@ -7,10 +7,13 @@ import { Logger } from "@server/loaders"
 
 import {
     MetadataAttributes,
+    Order,
+    Quote,
     Transaction as TransactionModel,
     TransactionAttributes,
     TransactionMetadata
 } from "../models"
+const { Op } = require("sequelize")
```
Comment (Extheoisah) [inline src/services/sequelize/repositories/transaction.ts]: why are we using require and not import statement here?
```suggestion
import { Op } from "sequelize"
```

File: src/services/sequelize/repositories/transaction.ts [hunk 3]
```diff
@@ -171,12 +187,82 @@ export const TransactionRepository = (): ITransactionRepository => {
...
+                },
+                include: [
+                    {
+                        model: TransactionMetadata,
+                        attributes: {
+                            exclude: [
+                                "orderId",
+                                "transactionId",
+                                "bankName",
+                                "bankAccountName",
```
Comment (Extheoisah) [inline src/services/sequelize/repositories/transaction.ts:215]: why are we excluding the bank details? if its a withdrawal, we need to show which bank account was paid

File: src/services/sequelize/repositories/transaction.ts (hunk 2 above)
Comment (tobi-bams) [reply src/services/sequelize/repositories/transaction.ts]: Fixed this already, was a silly mistake

File: src/services/sequelize/repositories/transaction.ts (hunk 3 above)
Comment (tobi-bams) [reply src/services/sequelize/repositories/transaction.ts:215]: Nothing specific I can put it back, I just did not see where we are using it on UI

File: src/server/routes/transaction.ts (hunk 1 above)
Comment (tobi-bams) [reply src/server/routes/transaction.ts:67]: Thanks

File: src/services/sequelize/repositories/transaction.ts (hunk 3 above)
Comment (Extheoisah) [reply src/services/sequelize/repositories/transaction.ts]: We would use it in the txn detail card. the modal that shows the full txn details

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**